dist/plugin.js

```diff
@@ -22,11 +22,16 @@
         eventName in this.options && this.options[eventName].call(this, $.Event(this.name + ':' + eventName, {bubbles: false}), data);
     };
 
-    Plugin.create = function(name, SubConstructor, prototype) {
-        SubConstructor.__super__ = Plugin;
-        for (var key in Plugin.prototype) {
+    Plugin.create = function(name, SubConstructor, SuperConstructor, prototype) {
+        if (typeof prototype === 'undefined') {
+            prototype = SuperConstructor;
+            SuperConstructor = Plugin;
+        }
+
+        SubConstructor.__super__ = SuperConstructor;
+        for (var key in SuperConstructor.prototype) {
             if (!SubConstructor.prototype[key]) {
```

Comment:
We can but it really doesn't matter, unless you can think of a use case where it would.